# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/tests/mochitest/general/performance_timeline_main_test.html
# Commit: 9a8abd87cc79
# Full Hash: 9a8abd87cc7935f29b94248c1a6f8203faa14403
# Author: Henri Sivonen <hsivonen@hsivonen.fi>
# Date: 2021-12-08 21:43:06
# Regressor Bug: 1701828
# File Overlap Count: 10
# Description:
#   Bug 1701828 - meta charset rewrite. r=smaug
#   
#   Implements https://github.com/whatwg/html/issues/6962 . Improves performance
#   when <meta charset> occurs in head but after the first kilobyte and aligns
#   behavior better with WebKit and Blink.
# ==============================================================================

diff --git a/dom/tests/mochitest/general/performance_timeline_main_test.html b/dom/tests/mochitest/general/performance_timeline_main_test.html
--- a/dom/tests/mochitest/general/performance_timeline_main_test.html
+++ b/dom/tests/mochitest/general/performance_timeline_main_test.html
@@ -1,16 +1,17 @@
 <!--
   Any copyright is dedicated to the Public Domain.
   http://creativecommons.org/publicdomain/zero/1.0/
 -->
 
 <!DOCTYPE html>
 <html>
 <head>
+  <meta charset="utf-8">
 	<link rel="stylesheet" href="/tests/SimpleTest/test.css?performance-timeline-main-test"/>
 	<script type="application/javascript">
 
 function ok(cond, message) {
   window.opener.ok(cond, message)
 }
 
 function is(received, expected, message) {